cmd/notary/integration_test.go

```diff
@@ -2125,7 +2131,7 @@ func TestClientKeyImport(t *testing.T) {
 	// if there is hardware available, root will only be on hardware, and not
```

Comment:
That setup op top can be factored out as a helper function, and return the temp files instead  :)  Up to you though.